web-ui/src/components/agenda/Agenda.jsx

```diff
@@ -44,9 +44,10 @@ const AgendaItems = ({ checkinId, memberName }) => {
       if (res && res.payload) {
         let agendaItemList =
           res.payload.data && !res.error ? res.payload.data : undefined;
-        agendaItemList.sort((a, b) => {
-          return a.priority - b.priority;
-        });
+        agendaItemList &&
+          agendaItemList.sort((a, b) => {
+            return a.priority - b.priority;
+          });
         setAgendaItems(agendaItemList);
```

Comment:
This makes sonar happy, but I'm not sure it's actually doing what you want. I think you probably want the null check to effect whether or not you even call `setAgendaItems`, don't you?